### DISCUSSION TITLE
Geração do token do google - Como fazer sem ser pelo projeto angular?

### DISCUSSION THREAD

**Discussion by angelosilva-dev** [balta-io/forum#368] [Tira Dúvidas] [answered]
**Curso**: Todo App: API com ASP.NET Core, CQRS e EF Core
**Aula**:  Obtendo Token do Google

Na aula Obtendo token do google do curso citado acima, é ensinado a gerar o token do google para utilizar no postman para autenticar os endpoints da API.
No exmeplo da aula, é utilizado uma aplicação em angular para realizar a geração do token, o que pra mim não deu certo devido ao npm install não conseguir instalar as dependencias por completo desse projeto angular, pois da um erro ao realizar o npm install.
Tentei instalar o C++ como sugerido em outro topico de duvidas que vi aqui, mas também não deu certo.
Portanto,  gostaria de saber uma outra maneira de gerar esse token para eu finalizar os teste da api Todo.

**Comment by andrebaltieri** [comment, marked as answer]
@angelosilva-dev como vai?
Acredito que o Token da parte do Angular não é o do Google e sim do Firebase, que no caso precisa da SDK do Firebase para ser gerado, por isto não sei se pode ser feito via Postman.

Você pode encontrar a documentação atualizada, caso esteja em uma versão nova do Angular aqui:
https://github.com/angular/angularfire
